modules/post/windows/gather/file_from_raw_ntfs.rb

```diff
@@ -16,7 +16,8 @@ def initialize(info = {})
       'Name'         => 'Windows File Gather File from Raw NTFS',
       'Description'  => %q(
           This module gathers a file using the raw NTFS device, bypassing some Windows restrictions
-          such as open file with write lock. Can be used to retrieve files such as NTDS.dit.),
+          such as open file with write lock. Because it avoids the usual file locking issues, it can
+          be used to retrieve files such as NTDS.dit.),
```

Comment:
Maybe make it a brace and then drop it down a line to be more consistent and less confusing?